share/picongpu/examples/ThermalTest/include/picongpu/ThermalTestSimulation.hpp

```diff
@@ -197,7 +197,7 @@ namespace picongpu
         //   you may like to let the plasma develope/thermalize a little bit
         static constexpr uint32_t firstTimestep = 1024;
 
-        container::HostBuffer<float, 2>* eField_zt[2];
+        container::HostBuffer<float, 2>* eField_zt[2] = {nullptr, nullptr};
```

Comment:
You could also use a `unique_ptr` here to modernize and tackle the problem.

Reply:
Yes, you are right, I was lazy because nobody is using the test anyway^^ I already thought about removing it. 
IMO it makes no sense to keep a dead test nobody is ever running. The test will not be lost because we have it since the beginning in the git history.

I will talk tomorrow with the other dev's about removing it.

Reply:
updated to `unqiue_ptr` but will still discuss this test in the dev meeting.